lib/runner/cursor.js

```diff
@@ -85,7 +85,7 @@ _.extend(Cursor.prototype, {
         (cycles == null) && (cycles = this.cycles);
 
         // use the constructor to set the values
-        Cursor.call(this, length, cycles, this.position, this.iteration);
+        Cursor.call(this, length, cycles, this.position, this.iteration, this.ref);
```

Comment:
bounds wound retain ref. as such Cursor.call retains ref. You do not need to pass this.